docs/BigQueryDropDataset-action.md

```diff
@@ -0,0 +1,34 @@
+# BigQuery Drop Dataset
+
+#### **Description**
+
+Drop a BigQuery dataset.
+
+#### **Basic**
+
+Following are properties used to configure this plugin
+
+* **Dataset**
+
+  Name of dataset to be dropped. Name may contain up to 1,024
```

Comment:
Name of the dataset to drop. The name may contain up to 1,024 characters and must consist of letters (upper or lower case), numbers, or underscores.